src/main/java/com/github/manolo8/darkbot/config/Config.java

```diff
@@ -227,6 +225,12 @@
             public @Option @Num(max = 300, step = 1) int TRAIL_LENGTH = 15;
             public @Option boolean MAP_START_STOP = false;
             public @Option(key = "colors") ColorScheme cs = new ColorScheme();
+            public @Option boolean CUSTOM_BACKGROUND = false;
+            public @Option @Editor(JPercentField.class) double TRANSPARENCY = 0.3f;
+            @Option
+            @Editor(value=JFileOpener.class, shared = true)
+            public String CHOOSER = "";
```

Comment:
You're gonna need to make an editor for specifically images, make it either extend JFileOpener or be similar (mostly copy-pasted from JFileOpener).
Make the type it saves to be the ImageWrapper defined in an earlier comment (https://github.com/darkbot-reloaded/DarkBot/pull/118/files#r775158000), make it so the editor calls setPath when a user picks a different file in the file chooser, and call getImage from map drawer